tests/sklearn/test_sklearn_models.py

```diff
@@ -151,6 +151,28 @@ def get_n_bits_non_correctness(model_class):
     return n_bits
 
 
+def fit_and_compile(model, x, y):
+    """Fit the model and compile it."""
+
+    with warnings.catch_warnings():
+        # Sometimes, we miss convergence, which is not a problem for our test
+        warnings.simplefilter("ignore", category=ConvergenceWarning)
+        model.fit(x, y)
+
+    model.compile(x)
+
+
+def get_random_samples(x, n_sample):
```

Comment:
small detail but looks like something to put in pytest utils !

Reply:
Are you sure ?
It's only used for sklearn tests

Reply:
I think it's something we might want to do for different kinds of tests, so it may be a good idea to move it yes ! but it's not very important no worries

Reply:
moved to pytest folder